src/DynamoUtilities/PathHelper.cs

```diff
@@ -102,6 +102,46 @@ public static bool HasWritePermissionOnDir(string folderPath)
             }
         }
 
+        /// <summary>
+        /// Returns whether current user has read access to the folder path.
+        /// </summary>
+        /// <param name="folderPath">Folder path</param>
+        /// <returns></returns>
+        internal static bool HasReadPermissionOnDir(string folderPath)
```

Comment:
hmm, can this code be simplified by simply trying to read the location and catching?

Reply:
sorry missed this was added to the utils, seems good.